frontend/src/components/regions/CardDetailsModal/ReviewsSbonelo.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
```

Comment:
Should this file exist? 

And if it should exist why does it have such a fully name?  Please name things properly. 

If I ask you "what does this component?" You will say "It is a card that displays a single review". So "ReviewCard" is a better name. 

This component does not draw multiple reviews. So having a pleural in the name is not appropriate.
This component is also not a Sbonelo.

If we need to see who wrote the code, we can use git.

Reply:
This has definitely been removed. It was for practice, I pushed the code for pushing daily